Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/security.yml

```diff
@@ -65,3 +65,15 @@
     contents:
     - id: WeaponEnergyGunMini
       amount: 3
+
+- type: entity
```

Comment:
this should either be upstreamed or in its own file, and what colin said

Reply:
I mean the M90 is a DeltaV gun, isn't it supposed to be with the rest of DeltaV gun safe fills?

Reply:
oh i cant read

Reply:
![obraz_2024-07-07_135800440](https://github.com/DeltaV-Station/Delta-v/assets/96891493/944cc6fe-e633-424b-b59e-d9bb45b773e9)